k8s.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
This is confusing: what is this file for?

Reply:
It's a configuration file in order to run the operator inside the Okteto pod. I will remove it as we are removing this dependency.